crates/bevy_ecs/src/lib.rs

```diff
@@ -39,9 +39,9 @@ pub mod prelude {
             StageLabel, State, SystemLabel, SystemSet, SystemStage,
         },
         system::{
-            Commands, In, IntoChainSystem, IntoExclusiveSystem, IntoSystem, Local, NonSend,
-            NonSendMut, ParallelCommands, ParamSet, Query, RemovedComponents, Res, ResMut,
-            Resource, System, SystemParamFunction,
+            adapter as system_adapter, Commands, In, IntoChainSystem, IntoExclusiveSystem,
```

Comment:
```suggestion
            adapter as chain_adapter, Commands, In, IntoChainSystem, IntoExclusiveSystem,
```
Wouldn't it be better to use `chain_adapter` as those adapters can only be used on `ChainSystem`s?

Reply:
The current plan is to rename `.chain()` to `.pipe()`, so I'd rather not give it a name that we'd have to change later.

Reply:
Make sense